src/plonetheme/eeq/theme/rules.xml

```diff
@@ -175,4 +176,18 @@
   <drop css:theme="#portal-toolbar" css:if-content=".userrole-anonymous" />
   <replace css:theme="#anonymous-actions" css:content-children="#portal-personaltools-wrapper" css:if-not-content=".ajax_load" css:if-content=".userrole-anonymous" />
 
+  <!-- Link normalization -->
+  <before css:theme-children="a.navbar-brand"><xsl:attribute name="href"><xsl:value-of select="$portal_url" />/</xsl:attribute></before>
+  <before css:theme-children="a.report-bias"><xsl:attribute name="href"><xsl:value-of select="$portal_url" />/report-bias</xsl:attribute></before>
+  <before css:theme-children="a.give-now"><xsl:attribute name="href"><xsl:value-of select="$portal_url" />/giving</xsl:attribute></before>
+  <before css:theme-children="a.contact-button"><xsl:attribute name="href"><xsl:value-of select="$portal_url" />/contact-info</xsl:attribute></before>
```

Comment:
This is fine, but I think merging classes and copying the attributes from the existing action links in the content might be a better approach generally.

Reply:
Ok. Noted